bitnami/mongodb/templates/arbiter/statefulset.yaml

```diff
@@ -145,11 +145,11 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: K8S_SERVICE_NAME
-              value: "{{ include "mongodb.fullname" . }}-arbiter-headless"
+              value: "{{ include "mongodb.arbiterservicename" . }}"
```

Comment:
Change arbiterservicename to arbiter.service.nameOverride

Reply:
Fixed